src/fparser/two/utils.py

```diff
@@ -1308,69 +1308,87 @@ class WORDClsBase(Base):
 
     '''
     @staticmethod
-    def match(pattern, cls, string, check_colons=False, require_cls=False):
-        ''':param pattern: the pattern of the WORD to match. This can be a \
-        string, a list of strings or a tuple of strings.
-        :type pattern: str, tuple of str or list of str.
+    def match(pattern, cls, string, colons=False, require_cls=False):
```

Comment:
pylint complains that `pattern` is being re-defined from outer scope.

Reply:
Oops. Changed to keyword. Also fixed pylint error about return values.